packages/cli/src/cli.ts

```diff
@@ -192,6 +196,15 @@ build({
     useSourceNamespace: useNamespaceReference,
 });
 
+if (diagnosticsMessages.length) {
+    if (diagnostics) {
+        console.log('[Stylable Diagnostics]\n', diagnosticsMessages.join('\n\n'));
+    }
+    if (!ignoreDiagnostics) {
```

Comment:
Should we be more flexible in how we treat errors vs. warnings?

Reply:
yes.